cocos/scripting/js-bindings/jswrapper/v8/ScriptEngine.cpp

```diff
@@ -707,11 +713,171 @@ namespace se {
         return _fileOperationDelegate;
     }
 
+
+    bool ScriptEngine::saveByteCodeToFile(const std::string& path, const std::string& path_bc)
+    {
+        bool success = false;
+        auto fu = cocos2d::FileUtils::getInstance();
+
+        if (path_bc.length() > 3 && path_bc.substr(path_bc.length() - 3) != ".bc") {
+            SE_LOGE("ScriptEngine::generateByteCode bytecode file path should endwith \".bc\"\n");;
+            return false;
+        }
+
+        if (fu->isFileExist(path_bc))
+        {
+            SE_LOGE("ScriptEngine::generateByteCode file already exists!\n");;
+            return false;
+        }
```

Comment:
```suggestion
        if (fu->isFileExist(path_bc))
        {
            SE_LOGE("ScriptEngine::generateByteCode file already exists, it will be rewrite!\n");
        }
```